src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ParseRequestService.java

```diff
@@ -29,24 +36,29 @@ public void validateUserProfiles(List<UserProfile> userProfiles, UserRequest use
             throw new ResourceNotFoundException("The user profiles couldn't be found");
         }
```

Comment:
this can also be converted in to error message instead of runtime exception

Reply:
This is also the dead point as beyond this no meaning to proceed the flow.